src/main/kotlin/org/jetbrains/kotlin/jupyter/repl.kt

```diff
@@ -88,9 +88,10 @@ class ReplForJupyter(val scriptClasspath: List<File> = emptyList(),
             val kt = KotlinType(receiver.javaClass.canonicalName)
             implicitReceivers.invoke(listOf(kt))
 
-            val classes = listOf(/*receiver.javaClass,*/ ScriptTemplateWithDisplayHelpers::class.java)
-            val classPath = classes.asSequence().map { it.protectionDomain.codeSource.location.path }.joinToString(":")
-            compilerOptions.invoke(listOf("-classpath", classPath, "-jvm-target", "1.8"))
+            //val classes = listOf(/*receiver.javaClass,*/ ScriptTemplateWithDisplayHelpers::class.java)
+            //val classPath = classes.asSequence().map { it.protectionDomain.codeSource.location.path }.joinToString(":")
```

Comment:
Are these comments still needed? Where did this logic go?

Reply:
Additional classpath is not needed, because shared classes for completion have already been moved to jupyter-lib module that is isolated from kernel internals. -Jvm-target needs to be restored, thanks for pointing this out

Reply:
-jvm-target is already applied below this code. So, comment can be removed